### PR TITLE
New Site

### PR DESCRIPTION
**Pull request by benmcgarry**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+- 2.1
+- 2.2
+# Assume bundler is being used, therefore
+# the `install` step will run `bundle install` by default.
+install: gem install jekyll rdiscount pygments.rb #html-proofer
+script: jekyll build #&& htmlproof ./_site --disable-external --only-4xx --check-html --check-favicon
+# branch whitelist, only for GitHub Pages
+branches:
+  only:
+  - gh-pages     # test the gh-pages branch
```
Comment (erikdesjardins) [inline .travis.yml:13]: Probably needs to be `master`, since this is a special repo where the master branch is the site (I think)

Comment (erikdesjardins) [inline .travis.yml:13]: Or maybe we don't need it at all

Comment (jewel-andraia) [inline .travis.yml:13]: I seem to be missing the magic involved in the deploy step. Is that handled as a repo trigger?

Comment (erikdesjardins) [inline .travis.yml:13]: I believe it's all hidden and unconfigurable. GitHub may even render the site live when you request it.

I think this travis config was just for testing.

**Pull request merged**